Apromore-Core-Components/Apromore-BPMN-Editor/Apromore-BPMN-Modeler-JS/app/translate/translations-ja.js

```diff
@@ -90,5 +90,7 @@ module.exports = {
   'invalid.endTime {beginTime}':	'無効な終了時間：{beginTime}以降の時間を選択してください。',
 
   'details':	'詳細',
-  'N/A':	'該当なし'
+  'N/A':	'該当なし',
+
+  'properties': 'Properties'
```

Comment:
Do we have a procedure for ensuring that new localizations like this one are eventually translated?

Reply:
We're still doing this manual check through excel master sheet in OneDrive.